### ISSUE TITLE
FormItem position issues when they are not visible at initialization!

### ISSUE THREAD

**Issue by sandronm** [vaadin/vaadin-form-layout-flow#102]
This ticket is similar to the following: https://github.com/vaadin/vaadin-form-layout-flow/issues/92
Except the fix is not fixing the issue (tested by using the last version of vaadin-form-layout 2.1.3).

Here is the code:

```
		FormLayout formLayout = new FormLayout();
		formLayout.setResponsiveSteps(new FormLayout.ResponsiveStep("1px", 1, FormLayout.ResponsiveStep.LabelsPosition.TOP));

		FormLayout.FormItem nameFormItem = formLayout.addFormItem(new TextField(), "Name");
		FormLayout.FormItem lastnameFormItem = formLayout.addFormItem(new TextField(), "Lastname");
		FormLayout.FormItem addressFormItem = formLayout.addFormItem(new TextArea(), "Address");
		FormLayout.FormItem birthdayFormItem = formLayout.addFormItem(new DatePicker(), "Birthday");

		Button button = new Button("Toogle visibility of some fields");
		button.addClickListener(event -> {
			addressFormItem.setVisible(!addressFormItem.isVisible());
			birthdayFormItem.setVisible(!birthdayFormItem.isVisible());
		});

		addressFormItem.setVisible(false);
		birthdayFormItem.setVisible(false);

		add(button, formLayout);

```
Please note the 2 last lines: **address and birthday FormItems must be by default invisible.**

When I press the button to make them visible, here is the result:

![image](https://user-images.githubusercontent.com/17686013/59667991-fa8b8100-91b7-11e9-8005-7d44e0b53d93.png)

But if I remove the 2 last lines (then by default, all my fields are visible) and if I click to make them invisible and click again to make them again visible, there is no issue...

![image](https://user-images.githubusercontent.com/17686013/59668304-87ced580-91b8-11e9-9d23-7894329e5a2b.png)

@Haprog could you please check this one?

**Comment by Haprog** [comment]
@sandronm After toggling the visibility of items in the `FormLayout` you can fix the styles by running:
```java
formLayout.getElement().callFunction("updateStyles");
```

So in the above example it work if you update the click handler to this:
```java
button.addClickListener(event -> {
    addressFormItem.setVisible(!addressFormItem.isVisible());
    birthdayFormItem.setVisible(!birthdayFormItem.isVisible());
    formLayout.getElement().callFunction("updateStyles");
});
```

I'm not sure if it's possible to make this work automatically in any clean way or should we just add this info to the documentation. Maybe we could at least we could provide a helper method in the Java API like `formLayout.updateStyles()` to make this a bit easier.

**Comment by web-padawan** [comment]
@sandronm just to clarify, what browser version are you using? Is that IE11?

**Comment by sandronm** [comment]
Thank you, it works indeed when I call the method "updateStyles" but yes, probably more documentation is needed. Anyway, this improvement https://github.com/vaadin/vaadin-form-layout/issues/116 will be appreciated

**Comment by Haprog** [comment]
We could consider adding a demo that shows using `updateStyles` with dynamically shown/hidden items.

**Comment by sandronm** [comment]
Yes please :-)

Thank you for your work, you may close this issue if you want.

**Comment by web-padawan** [comment]
Closing per above comment, let's continue with the actual fix at vaadin/vaadin-form-layout#116
